CRITICAL - 2023-04-10 08:11:43 --> Call to undefined function form_open()
in APPPATH\Views\user_view.php on line 58.
 1 SYSTEMPATH\View\View.php(213): include()
 2 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 3 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('user_view', [], true)
 4 APPPATH\Views\layout\conten.php(3): view('user_view')
 5 APPPATH\Views\layout\wrapper.php(4): require_once('APPPATH\\Views\\layout\\conten.php')
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\wrapper.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('layout/wrapper', [], true)
 9 APPPATH\Controllers\User.php(15): view('layout/wrapper', [...])
10 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->index()
11 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
12 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-04-10 08:25:55 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\Portal_berita\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Portal_berita\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Portal_berita\app\Models\User_model.php(15): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Portal_berita\app\Controllers\User.php(13): App\Models\User_model->listing()
#7 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(934): App\Controllers\User->index()
#8 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Portal_berita\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Portal_berita\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\Portal_berita\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\Portal_berita\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\Portal_berita\app\Models\User_model.php(15): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\Portal_berita\app\Controllers\User.php(13): App\Models\User_model->listing()
#6 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(934): App\Controllers\User->index()
#7 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#8 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\Portal_berita\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\Portal_berita\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-04-10 08:25:55 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 3 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Models\User_model.php(15): CodeIgniter\Model->__call('get', [])
 5 APPPATH\Controllers\User.php(13): App\Models\User_model->listing()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-04-10 08:26:21 --> mysqli_sql_exception: Table 'db_berita.users' doesn't exist in C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Portal_berita\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Portal_berita\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Portal_berita\app\Models\User_model.php(15): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Portal_berita\app\Controllers\User.php(13): App\Models\User_model->listing()
#7 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(934): App\Controllers\User->index()
#8 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Portal_berita\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Portal_berita\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-10 08:26:21 --> Table 'db_berita.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 2 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\User_model.php(15): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\User.php(13): App\Models\User_model->listing()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-04-10 08:26:22 --> mysqli_sql_exception: Table 'db_berita.users' doesn't exist in C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\Portal_berita\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\Portal_berita\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\Portal_berita\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\Portal_berita\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\Portal_berita\app\Models\User_model.php(15): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\Portal_berita\app\Controllers\User.php(13): App\Models\User_model->listing()
#7 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(934): App\Controllers\User->index()
#8 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
#9 C:\xampp\htdocs\Portal_berita\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\Portal_berita\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\Portal_berita\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}
CRITICAL - 2023-04-10 08:26:22 --> Table 'db_berita.users' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `users`', [], false)
 2 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 3 APPPATH\Models\User_model.php(15): CodeIgniter\Model->__call('get', [])
 4 APPPATH\Controllers\User.php(13): App\Models\User_model->listing()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
CRITICAL - 2023-04-10 08:29:50 --> syntax error, unexpected 'else' (T_ELSE)
in APPPATH\Views\user_view.php on line 44.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('user_view', [], true)
 3 APPPATH\Views\layout\conten.php(3): view('user_view')
 4 APPPATH\Views\layout\wrapper.php(4): require_once('APPPATH\\Views\\layout\\conten.php')
 5 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\layout\\wrapper.php')
 6 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 7 SYSTEMPATH\Common.php(1184): CodeIgniter\View\View->render('layout/wrapper', [], true)
 8 APPPATH\Controllers\User.php(19): view('layout/wrapper', [...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\User->index()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\User))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
